### PR TITLE
lk86/update-connect-to-berkeley 

### PR DESCRIPTION
**Pull request by lk86**
Update connect to berkeley test to add libp2p keys and to run the same way on the develop branch

### CODE REVIEW

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

File: buildkite/scripts/connect-to-berkeley.sh [hunk 1]
```diff
@@ -30,9 +30,17 @@ apt-get install --allow-downgrades -y "mina-${TESTNET_NAME}=${MINA_DEB_VERSION}"
 # Remove lockfile if present
 rm ~/.mina-config/.mina-lock ||:
 
+mkdir -p /root/libp2p-keys/
+# Pre-generated random password for this quick test
+export MINA_LIBP2P_PASS=eithohShieshichoh8uaJ5iefo1reiRudaekohG7AeCeib4XuneDet2uGhu7lahf
```
Comment (QuiteStochastic) [inline buildkite/scripts/connect-to-berkeley.sh:35]: not super familiar with this tool but it looks like you're setting the password to the key before you're even generating the key?  how does that work?

**Review by QuiteStochastic — APPROVED** [review]

**Review by deepthiskumar — APPROVED** [review]

**Review by stevenplatt — APPROVED** [review]
approved.

**Pull request merged**